Dom-mainpulation.js

```diff
@@ -1,14 +1,481 @@
+// Frist window
+
 function mobileMenu() {
-  if (document.getElementById('Purple').style.display === 'none') {
-    document.getElementById('Purple').style.display = 'flex';
-  } else {
-    document.getElementById('Purple').style.display = 'none';
-  }
+  const window = document.querySelector('.Purple');
+  window.classList.replace('not-active', 'active');
+}
+
+function closeMenu() {
+  const window = document.querySelector('.Purple');
+  window.classList.replace('active', 'not-active');
 }
+
 document.getElementById('hamb-btn').addEventListener('click', mobileMenu);
-document.getElementById('close').addEventListener('click', mobileMenu);
+document.getElementById('close').addEventListener('click', closeMenu);
 const anchorElm = document.querySelectorAll('.pop-window');
 
 for (let i = 0; i < anchorElm.length; i += 1) {
-  anchorElm[i].addEventListener('click', mobileMenu);
+  anchorElm[i].addEventListener('click', closeMenu);
 }
+
+// Second window
+const bodyDiv = document.querySelector('.pop-body');
+
+// First card
+function addModal1() {
```

Comment:
- [ ] Your popup functionality seems not to be working, When I click the popup button, the popup didn't display. Kindly check this out 👍🏽

Reply:
when I click on the seeProject button the event is activated as you can see in this pictures, 
so I don't really know what the problem could be, maybe something wrong with your browser ?

![cap 1](https://user-images.githubusercontent.com/24640848/195487291-d5b927a9-5dda-4550-bb0e-e548c1a9b6eb.png)
![cap 2](https://user-images.githubusercontent.com/24640848/195487337-fe2747d2-5044-4d2c-812e-85f7f40fbf23.png)